tensorboardX/writer.py

```diff
@@ -25,7 +25,7 @@
 
 try:
     from caffe2.proto import caffe2_pb2
-    from caffe2.python import cnn, core, workspace
+    from caffe2.python import cnn, core, model_helper, workspace
```

Comment:
Where do you use model_helper ?

Reply:
I used it to check if the model is an instance of model_helper.ModelHelper, but I later changed the check so it first checks whether it is a core.Net or caffe2_pb2.NetDef, then if it is neither, it just assumes it is a Caffe2 model. I can delete this?

Reply:
Yes, unnecessary import should be removed.